### ISSUE TITLE
[Request]: Make alert dialogs HIG compliant

### ISSUE THREAD

**Issue by jamiethecat** [GeopJr/Tuba#1036]
### Describe the request

Currently, the app tends to use sentences in titles when it comes to alert dialogs. The title should be title case and contain 2-4 words. Descriptions shouldn't use a period if they contain a single sentence.

### Implementation Details

- [X] This follows the [GNOME HIG](https://developer.gnome.org/hig/).

**Comment by GeopJr** [comment]
On the '2-4 words' part, not sure how to comply on some stuff, e.g.

"Are you sure you want to no longer federate with %s?", is definitely too wordy for a title, but also I have no idea what the title should be instead

Similarly, another edge case is:

title: "Are you sure you want to block %s?"
subtitle: "This can be the domain name that shows up in the e-mail address or the MX record it uses. They will be checked upon sign-up."

If you want to give it a try, I'll accept any changes you make, but I'm really bad at this lol

P.S. you can search for them in the source code by `Adw.AlertDialog` and `app.question` (<-- higher level api that uses alert dialogs)

**Comment by jamiethecat** [comment]
> "Are you sure you want to no longer federate with %s?", is definitely too wordy for a title, but also I have no idea what the title should be instead

"Stop Federating With %s?"

> "Are you sure you want to block %s?"

"Block %s?"

I'm a bit busy right now so I can't make the changes myself 😔

**Comment by GeopJr** [comment]
I see, thanks!

> I'm a bit busy right now so I can't make the changes myself 😔

No worries! (they'll have to wait for 0.9 anyway, so I'll go over them at a later point)

**Comment by GeopJr** [comment]
How should dialog titles that 100% match the button label be rewritten as?

E.g.

"Are you sure you want to take action?"
"Cancel", "Take Action"

Would become

"Take Action?"
"Cancel", "Take Action"

But sounds a bit repetitive. Should it be written in a different way or just go with it anyway?

Similarly for "Submit?", "Submit"

**Comment by jamiethecat** [comment]
I think those strings actually deserve some rewording, "take action" is pretty broad, I'd specify what action is actually being taken here. For "Submit?" I'd also clarify what you're submitting.

**Comment by GeopJr** [comment]
Thanks!

I went with "Submit Report?" and "Take Action Against \<user handle\>?". 'Take Action' is an admin dashboard term so I'll leave it as is as to not confuse admins and it's a collection of operations so there's not a single action I can mention unfortunately (/ it can be 'suspending' and account, but it can also be doing nothing but send an e-mail to them or a combination of other options).
